diff --git a/data/systemd/fedora/groonga-httpd.service b/data/systemd/fedora/groonga-httpd.service
index 96129dc..fb13ec4 100644
--- a/data/systemd/fedora/groonga-httpd.service
+++ b/data/systemd/fedora/groonga-httpd.service
@@ -7,6 +7,7 @@ Conflicts=groonga.service
 Type=forking
 User=root
 Group=root
+ExecStartPre=-/usr/bin/mkdir -p /var/log/groonga/httpd
 ExecStart=/usr/sbin/groonga-httpd
 ExecStop=/usr/sbin/groonga-httpd -s stop
 
