--- groonga-1.0.2.orig/data/init.d/redhat/groonga	2010-09-09 14:53:26.000000000 +0900
+++ groonga-1.0.2/data/init.d/redhat/groonga	2010-10-06 13:52:21.319175965 +0900
@@ -2,7 +2,7 @@
 #
 # /etc/rc.d/init.d/groonga
 #
-# chkconfig: 2345 80 20
+# chkconfig: - 80 20
 # description: A full-text search engine and column store.
 # processname: groonga
 # config: /etc/sysconfig/groonga
@@ -10,7 +10,6 @@
 #
 ### BEGIN INIT INFO
 # Provides:          groonga
-# Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
 # Short-Description: groonga's init script
 # Description:       groonga is a full-text search engine and column store.
@@ -50,13 +49,6 @@
 	fi
 	DAEMON_ARGS="${DAEMON_ARGS} --user ${USER}"
 fi
-if [ -n "${GROUP}" ]; then
-	if ! getent group | grep -q "^${GROUP}:"; then
-		echo "$0: group for running $prog doesn't exist: ${GROUP}" >&2
-		exit 1
-	fi
-	CGROUP_DAEMON="${CGROUP_DAEMON} ${GROUP}"
-fi
 
 if [ -n "${PIDFILE}" ]; then
 	mkdir -p $(dirname ${PIDFILE})
@@ -147,8 +139,11 @@
     condrestart)
 	[ -f /var/lock/subsys/$prog ] && restart || :
 	;;
+    status)
+	status $prog
+	;;
     *)
-	echo "Usage: $prog {start|stop|reload|restart|condrestart}"
+	echo "Usage: $prog {start|stop|reload|restart|condrestart|status}"
 	exit 1
 	;;
 esac
