diff -ru groonga-1.3.0.orig/bindings/php/groonga.c groonga-1.3.0/bindings/php/groonga.c
--- groonga-1.3.0.orig/bindings/php/groonga.c	2011-01-27 20:45:31.748650900 +0900
+++ groonga-1.3.0/bindings/php/groonga.c	2012-02-01 15:06:44.810633374 +0900
@@ -10,7 +10,7 @@
   grn_ctx_close(ctx);
 }
 
-function_entry groonga_functions[] = {
+zend_function_entry groonga_functions[] = {
   PHP_FE(grn_ctx_init        , grn_ctx_init_arg_info)
   PHP_FE(grn_ctx_close       , grn_ctx_close_arg_info)
   PHP_FE(grn_ctx_connect      , grn_ctx_connect_arg_info)
